hack/build-cross.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Could you use `#!/usr/bin/env bash`, please? I learnt recently that `/bin/bash` is shell shim on macOS, and to make use of "the real" `bash` you need to use it through `env`. Since it also helps on FreeBSD as well, I think it's a good move.